examples/workflow-test/scripts/repositories.ts

```diff
@@ -90,7 +90,7 @@ function buildClient(options: GlobalOptions): void {
 }
 
 function buildTheia(options: GlobalOptions): void {
-    exec('yarn', [], { cwd: repositoryFolder(options.folder, theiaRepository) });
+    exec('yarn && yarn browser build', [], { cwd: repositoryFolder(options.folder, theiaRepository) });
```

Comment:
```suggestion
    if(exec('yarn', [], { cwd: repositoryFolder(options.folder, theiaRepository)})) {
        exec('yarn', ['browser', 'build'], { cwd: repositoryFolder(options.folder, theiaRepository) });
    }
```
Otherwise it fails on my machine.

Reply:
Can you also fix in the command `theia-integration` the start part (line 204)

from

`exec('yarn', ['start:ws:debug'], { cwd: repositoryFolder(folder, theiaRepository) });`

to

`exec('yarn', ['browser', 'start:ws:debug'], { cwd: repositoryFolder(folder, theiaRepository) });`